### DISCUSSION TITLE
Help -- DbContext

### DISCUSSION THREAD

**Discussion by GabrielNalin** [andrebaltieri/AgendaEventos#75] [General]
Pessoal,
Para criar o DbContext, efetuei as alterações que estã anexas. Perguntas:
[DbContext.txt](https://github.com/andrebaltieri/AgendaEventos/files/6622999/DbContext.txt)

Estão corretas?
Preciso criar uma pasta DbContext dentro do "controler" e inserir essas alterações lá?

**Comment by phjb** [comment]
@GabrielNalin seu arquivo está correto ao meu ver, apenas uma questão que é por convenção utilizamos os nomes das *DbSet* no plural, como pode ser verificado nesse exemplo do @andrebaltieri  [7194](https://github.com/balta-io/7194/blob/master/Data/DataContext.cs).

Quanto a criar a pasta *DbContext* dentro da *controller*, negativo, é criada uma pasta *Data* e seu _DbContext.cs_ fica dentro dela, como imagem abaixo: 
![image](https://user-images.githubusercontent.com/21105723/121343474-1b939200-c8f9-11eb-9210-ab2746f33544.png)

**Comment by GabrielNalin** [comment]
Muito obrigado, Paulo.

Vou corrigir e mandar pro git

Em qua, 9 de jun de 2021 08:05, Paulo Barbosa ***@***.***>
escreveu:

**Comment by GabrielNalin** [comment]
DataContext criada.

**Reply by JoaoNascSilva** [reply]
Bom dia @GabrielNalin , pelo que notei você criou no repo do @LuanFreitasRibeiro .
Passa para este repo aqui conforme recomendado nas reuniões anteriores e criando uma branch para a sua implementação e após feito esse processo faça um pull request solicitando review dos participantes.

Qualquer dúvida só perguntar o/.
